lib/tls.js

```diff
@@ -189,8 +183,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
   if (!valid) {
     var err = new Error(
-        util.format('Hostname/IP doesn\'t match certificate\'s altnames: %j',
-                    reason));
+        `Hostname/IP doesn't match certificate's altnames: "${reason}"`);
```

Comment:
JSON.stringify does offer protection for circular references... I don't think ```reason``` would cause that, but would appreciate a +1 that we don't have to worry about that.

As an aside, how do backticks handle circular refs?